product-service/getProduct.js

```diff
@@ -1,16 +1,29 @@
-const products = require('products-mock');
-const { successResponse, notFoundResponse, unexpectedErrorResponse } = require('helper');
+const { Client } = require('pg');
 
-module.exports = async (event) => {
+const { successResponse, notFoundResponse, unexpectedErrorResponse, validationErrorResponse } = require('helper');
+const { dbOptions } = require('./db/config');
+
+module.exports = async ( event ) => {
+    const client = new Client(dbOptions);
     try {
+        await client.connect();
+        console.log('getProduct handler input:', event);
         const { pathParameters } = event;
         const { id } = pathParameters;
-        const product = products.filter(elem => elem.id === id)[0];
-        if( !product ){
+        const { rows: products } = await client.query(`select products.id, products.title, products.price, products.description, stocks.count from products left join stocks on products.id = stocks.product_id where products.id='${id}';`);
+        if( !products[0] ){
             return notFoundResponse('product');
         }
-        return successResponse(product);
+        return successResponse(products[0]);
     }catch(e){
-        return unexpectedErrorResponse('product');
+        console.log('DB error: ', e)
+
+        if((e.code && e.code === '22P02') ){
```

Comment:
something strange here.
22P02 - what is it? At least it should be well-named constant